tests/ixia/bgp/helper.py

```diff
@@ -0,0 +1,255 @@
+    def getAvgDPDPConvergenceTime(portName,iter):
+        table,avg=[],[]
+        for i in range(0,iter):
+            logger.info('|---- {} Link Flap Iteration : {} ----|'.format(portName,i+1))
+            
+            #Start Traffic
+            logger.info('Starting Traffic')
+            ts = snappi_api.transmit_state()
+            ts.state = ts.START
+            response=snappi_api.set_transmit_state(ts)
+            assert(len(response.errors)) == 0
+            time.sleep(30)
+            flow_stats=get_flow_stats(snappi_api)
+            tx_frame_rate = flow_stats[0].frames_tx_rate
+            assert tx_frame_rate != 0
+
+            #Link Flap
+            logger.info('Simulating Link Failure on {} link'.format(portName))
+            ls = snappi_api.link_state()
+            ls.port_names = [portName]
+            ls.state = ls.DOWN
+            response=snappi_api.set_link_state(ls)
+            assert(len(response.errors)) == 0
+            time.sleep(15)
+            assert is_port_rx_stopped(snappi_api,portName) == True
+            flow_stats=get_flow_stats(snappi_api)
+            tx_frame_rate = flow_stats[0].frames_tx_rate
+            assert tx_frame_rate != 0
+            logger.info(tx_frame_rate)
+            
+            # Stop traffic
+            logger.info('Stopping Traffic')
+            ts = snappi_api.transmit_state()
+            ts.state = ts.STOP
+            response=snappi_api.set_transmit_state(ts)
+            assert(len(response.errors)) == 0
+            time.sleep(5)
+            flow_stats=get_flow_stats(snappi_api)
+            tx_frames = flow_stats[0].frames_tx
+            rx_frames = sum([fs.frames_rx for fs in flow_stats])
+            
+            # Calculate DPDP Convergence
+            dp_convergence = (tx_frames - rx_frames) * 1000 / tx_frame_rate
+            logger.info("DP/DP Convergence Time: {} ms".format(int(dp_convergence)))  
+            avg.append(int(dp_convergence))
+            logger.info('Simulating Link Up on {} at the end of iteration {}'.format(portName,i+1))
+            
+            #Link up at the end of iteration
+            ls.state = ls.UP
+            response=snappi_api.set_link_state(ls)
+            assert(len(response.errors)) == 0
+        table.append('%s Link Failure'%portName)
+        table.append(iter)
+        table.append(mean(avg))
+        return table
+    table=[]
+    flap_iterations=1 
+    #Iterating link flap test on all the rx ports
+    #flap_iterations can be modified as per preference
+    for i in range(0,len(rx_port_names)):
+        table.append(getAvgDPDPConvergenceTime(rx_port_names[i],flap_iterations))
+    columns=['Event Name','Iterations','Avg Calculated DP/DP Convergence Time(ms)']
+    logger.info("\n%s" % tabulate(table,headers=columns,tablefmt="psql"))
+
+def cleanup_config(duthost,tgen_ports,port_count):
+    """
+    Cleaning up dut config at the end of the test
+    Args:
+        snappi_api (pytest fixture): Snappi API
```

Comment:
`snappi_api` Arg is not used but added as part of description

Reply:
removed the snappi_api description